source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs

```diff
@@ -68,7 +68,7 @@ private string CreateBatchResultsSql(Guid batchId)
 SELECT {string.Join(", ", SqlColumnNames)}
 FROM {_deltaTableOptions.SCHEMA_NAME}.{_deltaTableOptions.RESULT_TABLE_NAME}
 WHERE {ResultColumnNames.BatchId} = '{batchId}'
-ORDER BY time
+ORDER BY {ResultColumnNames.CalculationResultId}
```

Comment:
The time series points must be in correct order in the calculation result object.
```suggestion
ORDER BY {ResultColumnNames.CalculationResultId}, {ResultColumnNames.Time}
```

Reply:
How can/do we test this functionality?

Reply:
Yes, I will make an integration test in a separate PR